packages/@ember/octane-app-blueprint/files/.eslintrc.js

```diff
@@ -42,9 +42,10 @@ module.exports = {
         node: true
       },
       plugins: ['node'],
-      rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
-        // add your custom rules and overrides for node files here
-      })
+      extends: [
+        'eslint:recommended',
```

Comment:
Are you sure we have to add this one a second time? I would have expected them to merge together from the top level

Reply:
I'm not -- I'll double check

Reply:
it does inherit!